src/constants.js

```diff
@@ -48,6 +48,24 @@ export const systemsOsFilterConfiguration = (policies) => ([{
     }))
 }]);
 
+export const systemsOsMinorFilterConfiguration = (osMajorVersion, osMinorVersions) => ([{
+    type: conditionalFilterType.group,
+    label: 'Operating System',
+    filterString: (value) => ([Object.keys(value).map((majorVersion) => (
+        Object.keys(value[majorVersion]).map((minorVersion) => (
+            `os_major_version = ${ majorVersion } AND os_minor_version = ${ minorVersion }`
+        ))
+    )).flat().join(' OR ')]),
+    items: [osMajorVersion].map((majorVersion) => ({
+        label: `RHEL ${ majorVersion }.0`,
```

Comment:
It doesn't really make sense to show RHEL 7.0 and then each RHEL 7.Y version under that.

```suggestion
        label: `RHEL ${ majorVersion }`,
```